dictionary/Dictionary.js

```diff
@@ -45,6 +45,24 @@ export default class Dictionary {
 	 *         evaluated using the provided parameters.
 	 */
 	get(key, parameters = {}) {}
+
+	/**
+	 * Tests whether the specified localization phrase exists in dictionary
+	 *
+	 * @param {string} key The key identifying the localization phrase.
+	 * @return {boolean} {@code true} if the the key (phrase) exists.
+	 */
+	has(key) {}
+
+	/**
+	 * Get directory scope for specific key
```

Comment:
I assume you meant to write "Retrieves the localization scope denoted by the provided partial key. This may be either an object representing a sub-group of location phrase generators, or a single generator if the provided keys denotes a single localization phrase" :).